src/ai/backend/manager/models/vfolder.py

```diff
@@ -927,14 +929,19 @@ async def _delete():
                         "volume": volume_name,
                         "vfid": str(folder_id),
                     },
-                ):
+                ) as (_, resp):
                     pass
-            except aiohttp.ClientResponseError:
-                failed_deletion.append(vfolder_info)
+            except (VFolderOperationFailed, InvalidAPIParameters) as e:
+                if resp.status == 404:
+                    row_deletion_infos.append(vfolder_info)
```

Comment:
This indirect cross reference would confuse static analysis tools because `resp` may not be defined depending on the exact location where the error occurs. `e` already contains the status code as it inherits aiohttp's exception classes.